File: pclassic/2016s/PClassic2016Stubs/stubs/BattleOfJakku.java
import java.util.*;
import java.io.*;
class BattleOfJakku {
  public static void main(String[] args) throws FileNotFoundException {
    Scanner sc = new Scanner(new FileReader("BattleOfJakkuIN.txt"));
    while (sc.hasNext()) {
      String num = sc.nextLine();
      String[] num1 = sc.nextLine().split(" ");
      String[] num2 = sc.nextLine().split(" ");
      String op = sc.nextLine();
      String ans = jakku(Integer.parseInt(num), num1, num2, op.charAt(0));
      System.out.println(ans);
    }
  }
  //Fill out the body of this method
  public static String jakku(int N, String[] num1, String[] num2, char op) {
    return "";
  }
}

File: pclassic/2016s/PClassic2016Stubs/stubs/BattleOfJakku.py
#Fill out the body of this method
def true_operations(arraylength, array1, array2, operator):
    outputarray = []

    return outputarray

def parse_file_and_call_function():
    with open("BattleOfJakkuIN.txt", "r") as f:
        while True:
            arraylength = f.readline()
            array1 = f.readline().split(' ')
            array2 = f.readline().split(' ')
            operator = f.readline()
            x = true_operations(arraylength, array1, array2, operator)

            print (x)



if __name__ == "__main__":
    parse_file_and_call_function()